libs/domain/picking/src/components/customer-note/src/customer-note.component.ts

```diff
@@ -1,16 +1,27 @@
 import { resolve } from '@spryker-oryx/di';
 import { PickingListMixin } from '@spryker-oryx/picking';
 import { RouterService } from '@spryker-oryx/router';
-import { i18n } from '@spryker-oryx/utilities';
+import { asyncState, i18n, valueType } from '@spryker-oryx/utilities';
 import { html, LitElement, TemplateResult } from 'lit';
-import { tap } from 'rxjs';
+import { of, switchMap, tap } from 'rxjs';
 import { styles } from './customer-note.styles';
 
 export class CustomerNoteComponent extends PickingListMixin(LitElement) {
   static styles = styles;
 
   protected routerService = resolve(RouterService);
 
+  protected isStartPickingLoading$ = this.pickingList$.pipe(
```

Comment:
And we can update `PickingListMixin` and provide extraction of currently picking list id.

Add this to the mixin:
```
@asyncState()
protected pickingInProgressId = valueType(this.pickingListService.getPickingInProgressId());
```

and then you can manage buttons state based on the value:

```
<oryx-button ?loading=${this.pickingInProgressId === this.pickingListId or what is the correct name}>
        <button ?disabled=${!!this.pickingInProgressId}>
```

Reply:
Don't forget to update interfaces